tensorboard/webapp/feature_flag/store/feature_flag_store_config_provider.ts

```diff
@@ -0,0 +1,41 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {InjectionToken} from '@angular/core';
+import {StoreConfig} from '@ngrx/store';
+
+import {FeatureFlagState, initialState} from './feature_flag_types';
+
+// /** @typehack */ import * as _typeHackStore from '@ngrx/store';
+
+/**
+ * Injection token for providing feature flag StoreConfig.
+ */
+export const FEATURE_FLAG_STORE_CONFIG_TOKEN: InjectionToken<StoreConfig<
+  FeatureFlagState
+>> = new InjectionToken<StoreConfig<FeatureFlagState>>(
+  'Feature Flag Store Config'
```

Comment:
For consistency: `[Feature Flag] Store Config` (this pattern is from ngrx docs)

Reply:
Done.